.github/workflows/ut_pr.yml

```diff
@@ -0,0 +1,52 @@
+name: PR Unit Test Coverage
+on:
+  pull_request_target:
+    types: [ opened, synchronize,reopened ]
+
+jobs:
+  test-name:
+    runs-on: ubuntu-latest
+    name: Get Change Paths
+
+    steps:
+      - name: Setup Go environment
+        uses: actions/setup-go@v3.2.1
+        with:
+          go-version: '1.18'
+
+      - name: Get PR Changed Paths
+        uses: guguducken/change-paths-action@v0.0.2
+        id: ut-pr
+        with:
+          github-token: ${{ secrets.ACTION_TOKEN }}
+          ignore: "pkg/pb/,etc/,optools/,LICENSES/,cgo/,cmd/generate-config/test/,docs/,proto/,test/"

```

Comment:
we should not skip `pkg/pb`, non-auto generated code exists in this path.

Reply:
I've removed it